src/miner_keys.erl

```diff
@@ -37,7 +46,14 @@ keys({file, BaseDir}) ->
                key_slot => undefined,
                ecdh_fun => libp2p_crypto:mk_ecdh_fun(PrivKey0),
                sig_fun => libp2p_crypto:mk_sig_fun(PrivKey0),
-               onboarding_key => undefined
+               onboarding_key => case filelib:ensure_dir(OnboardingKey) of
```

Comment:
Same as above.. also DRY: Move the block of code into a get_onboarding_key function

Reply:
Will do.